Validation/SiTrackerPhase2V/python/Phase2OTValidateTrackingRecHit_cff.py

```diff
@@ -0,0 +1,10 @@
+import FWCore.ParameterSet.Config as cms
+
+from Validation.SiTrackerPhase2V.Phase2OTValidateTrackingRecHit_cfi import * 
+
+trackingRechitValidOT = Phase2OTValidateTrackingRecHit.clone()
+
+from Configuration.ProcessModifiers.premix_stage2_cff import premix_stage2
+premix_stage2.toModify(trackingRechitValidOT,
+    pixelSimLinkSrc = "mixData:PixelDigiSimLink",
```

Comment:
```suggestion
    pixelSimLinkSrc = "mixData:PixelDigiSimLink",
    phase2TrackerSimLinkSrc = "mixData:Phase2OTDigiSimLink"
```
to avoid:
```
----- Begin Fatal Exception 11-Jun-2021 12:04:28 CEST-----------------------
An exception of category 'ProductNotFound' occurred while
   [0] Processing  Event run: 1 lumi: 1 event: 1 stream: 0
   [1] Running path 'validation_step13'
   [2] Calling method for module Phase2OTValidateTrackingRecHit/'trackingRechitValidOT'
Exception Message:
Principal::getByToken: Found zero products matching all criteria
Looking for type: edm::DetSetVector<PixelDigiSimLink>
Looking for module label: simSiPixelDigis
Looking for productInstanceName: Tracker

   Additional Info:
      [a] If you wish to continue processing events after a ProductNotFound exception,
add "SkipEvent = cms.untracked.vstring('ProductNotFound')" to the "options" PSet in the configuration.

----- End Fatal Exception -------------------------------------------------
```
in premixed workflows.

Reply:
done